elementary/monitor/api/groups/schema.py

```diff
@@ -0,0 +1,13 @@
+from typing import Dict, List, Union
+
+from pydantic import BaseModel
+
+DbtGroupSchema = Dict[str, Union[list, dict]]
+TagsGroupSchema = Dict[str, List[dict]]
+OwnersGroupSchema = Dict[str, List[dict]]
```

Comment:
In continue to my prev comment about having a schema for the grouped item - it will be really nice to be used here so we would easily know what are the values of the group schemas contain 🙂